tests/MongoDB.Driver.Core.TestHelpers/EventCapturer.cs

```diff
@@ -19,16 +19,16 @@
 using MongoDB.Bson;
 using MongoDB.Bson.IO;
 using MongoDB.Driver.Core.Events;
-using MongoDB.Driver.Core.Misc;
 
 namespace MongoDB.Driver.Core
 {
     public class EventCapturer : IEventSubscriber
     {
+        private Action<IEnumerable<object>, object> _addEventAction;
         private readonly Queue<object> _capturedEvents;
-        private readonly object _lock = new object();
-        private readonly IEventSubscriber _subscriber;
         private readonly Dictionary<Type, Func<object, bool>> _eventsToCapture;
+        private readonly IEventSubscriber _subscriber;
+        internal readonly object _lock = new object();
```

Comment:
this change (together with the line `27` and `80`) is required for a new class `EventNotifier`. 

I don't really like the changes in these lines, but it's the best which I could imagine for now. I will try to think about these lines tomorrow once again